### PR TITLE
chore: remove variable feature prompt from create variable prompts, prompt manually in create command

### PR DESCRIPTION
**Pull request by jsalaber**
# Changes

- added prompt before prompting to attach to a feature to ask user if they want to attach variable to a feature

### CODE REVIEW

**Comment by jsalaber** [comment]
Current dependencies on/for this PR:
* main
  * **PR #281** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/cli/281" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/DevCycleHQ/cli/281?utm_source=stack-comment).

File: src/commands/variables/create.ts [hunk 1]
```diff
@@ -38,7 +38,19 @@ export default class CreateVariable extends CreateCommand {
         }
         flags._feature = feature
 
-        const params = await this.populateParametersWithZod(CreateVariableDto, this.prompts, flags)
+        let params = await this.populateParametersWithZod(CreateVariableDto, this.prompts, flags)
+        if (!headless) {
+            const { attachToFeature } = await inquirer.prompt([{
+                name: 'attachToFeature',
+                message: 'Attach to a feature?',
```
Comment (ajwootto) [inline src/commands/variables/create.ts:45]: is this the clearest way to phrase this? Should we use similar language to the dashboard like "associate"?

maybe explain what attaching a variable to a feature will do for you in case you didn't know?

like

`Would you like to associate this variable to a feature? Doing so will allow you to serve different values of the variable to different users"

Comment (jsalaber) [reply src/commands/variables/create.ts:45]: good point, I think associate works better

Comment (elliotCamblor) [inline src/commands/variables/create.ts:45]: Do you think it would make sense for `attach to` to instead be `associate with`? Only because we use the term "associated" in the dashboard/docs, otherwise I wouldnt pick this nit

Comment (jsalaber) [reply src/commands/variables/create.ts:45]: yes I'll switch to associate

Comment (jsalaber) [reply src/commands/variables/create.ts:45]: although giving a more detailed explanation would be weird to do it just for this one spot, and might also just be too wordy

**Review by elliotCamblor — APPROVED** [review]
